ui/react-ui/package.json

```diff
@@ -12,6 +12,7 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "react-tooltip": "^4.2.13",
```

Comment:
I see you installed a new package for tooltips, which is good. We could also import bootstrap tooltips ;)

Reply:
I used bootstrap tooltip first, but it has some dependencies that apparently are deprecated and caused me to get a bunch of errors, so I moved to this one.